### DISCUSSION TITLE
Auto vacuum enable

### DISCUSSION THREAD

**Discussion by halokaya67** [Nozbe/WatermelonDB#1373] [Q&A]
Hello everyone,

I'm having some problems with my db because it grows and causes the app to get bigger but I want to stop it. I found that in sqlite3 we can enable auto_vacuum. But I don't know how to use it in watermelondb so I'd be happy if anybody can help me about how to enable auto_vacuum. Thanks in advance for your help.

**Comment by joaosodres** [comment]
I currently have this problem, how did you solve it?

**Reply by halokaya67** [reply]
I've used raw queries to vacuum every table after the db cleaned

**Comment by joaosodres** [comment]
Do you see any problem in doing this? apparently it worked for me
     
      await database.get('tableName').query(
        Q.unsafeSqlQuery(`VACUUM`)
      ).fetch();

**Reply by halokaya67** [reply]
No it looks absolutely fine glad that I could help mate... 👍😉

**Reply by joaosodres** [reply]
Thank you so much!
